### ISSUE TITLE
DataTables Responsive issue

### ISSUE THREAD

**Issue by projectmanagementsystem15** [daniel-nagy/md-data-table#346]
How to implement on angular material  + DataTables responsive design with plus icon. Like URL : http://l-lin.github.io/angular-datatables/#/withResponsive

**Comment by daniel-nagy** [comment]
Are you trying to add collapsable rows? Just put a second row with an `ngShow` directive. I believe something like this should work,

```html
<tr md-row ng-repeat-start="item in items">
  <td md-cell>
    <md-button class="md-icon-button" ng-click="showMore = ! showMore">
      <md-icon>add_circle</md-icon>
    </md-button>
  </td>
<tr md-row ng-repeat-end ng-show="showMore">
  <td></td>
</tr>
```

**Comment by projectmanagementsystem15** [comment]
Can you share complete example

**Comment by daniel-nagy** [comment]
This should get you started http://codepen.io/anon/pen/PNOZjg?editors=1010

**Comment by projectmanagementsystem15** [comment]
Thanks you for replay.

But we expect like responsive design like : https://datatables.net/ 

when we reduce browser size base on horizontal to hide a filed same time show plus icon
 
![responsive-1](https://cloud.githubusercontent.com/assets/18112925/14273830/1df2051e-fb2b-11e5-9327-a22b116def8a.png)

Please check it.

Thanks,
Yogesh Paghdal

**Comment by f1tech** [comment]
Yes, that should be relatively easy for you to implement based on the techniques I outlined.

**Comment by daniel-nagy** [comment]
You can use media queries to hide or show columns or other elements. You can use Angular Material's hide and show directives as well. Just be sure to hide corresponding cells in both the header and the body. For example, say you have 4 columns and you want to hide the 3rd column on small screens

```html
<table md-table>
  <thead md-head>
    <tr md-row>
      <th md-column></th>
      <th md-column></th>
      <th md-column hide-xs hide-ms show-gt-sm></th>
      <th md-column></th>
    </tr>
  </thead>
  <tbody md-body>
    <tr md-row>
      <td md-cell></td>
      <td md-cel></td>
      <td md-cel hide-xs hide-ms show-gt-sm></td>
      <td md-cel></td>
    </tr>
  </tbody>
</table>

**Comment by projectmanagementsystem15** [comment]
Great. but how can you check horizontal scroll base add "<th md-column hide-xs hide-ms show-gt-sm></th>" Can you share example

**Comment by f1tech** [comment]
https://material.angularjs.org/latest/layout/options

**Comment by projectmanagementsystem15** [comment]
good, if we have multiple table one table 3 Filed and second table 10 filed and third table 6 filed base on we can not use to like "https://material.angularjs.org/latest/layout/options" 

If you try to implement then you bettor understand of issue.

**Comment by gillirabbi** [comment]
@daniel-nagy I'm currently using the latest version of md-data-table and trying to implement a collapsable child row, for selected rows - like your Codepen (http://codepen.io/anon/pen/PNOZjg?editors=1010). However, on row selection the child rows appear below each table entry, i.e. each parent row. But in the Codepen demo, you're able to toggle the child row only for the selected row.

The code is almost identical to your Codepen, with ng-repeat-start/ng-repeat-end and a ng-show boolean value for the child row - which is set on select/deselect row. Any advice ?
